pages/1.10/installing/ent/custom/configuration/configuration-parameters/index.md

```diff
@@ -89,6 +90,28 @@ This topic provides configuration parameters available for [DC/OS Enterprise](ht
 | [zk_master_credentials](#zk-master)          | [enterprise type="inline" size="small" /] The ZooKeeper master credentials.  |
 | [zk_agent_credentials](#zk-agent)           | [enterprise type="inline" size="small" /] The ZooKeeper agent credentials.  |
 
+[enterprise]
+### adminrouter_auth_cache_enabled
+[/enterprise]
+
+_This option was Added in DC/OS 1.10.6._
+
+Controls whether the Admin Router authorization cache is enabled.
+
+Most DC/OS authorizers (Mesos, Marathon, etc.) cache a user's permissions for 5 seconds after performing an authorization check for that user.
+This prevents users who issue bursts of requests from overloading the IAM.
+To date Admin Router did not implement caching of permissions.
+This meant that every request to certain endpoints through Admin Router triggered an authorization check to ensure that the user has sufficient permissions to do so.
+While this works well for typical human interaction with a DC/OS cluster we found that applications connecting to the cluster through Admin Router often issued many requests in the space of a fewf seconds.
+Such a high volume of requests, each leading to a separate authorization check, easily placed too much load on the IAM.
+With the IAM overloaded its performance would degrade and authorization checks throughout the cluster would begin to timeout.
+The solution was to give Admin Router a short-lived authorization cache similar to that implemented by Mesos and Marathon.
+As some users may rely on Admin Router not caching permissions we decided not to enable this feature by default until the next major DC/OS release, namely DC/OS 1.12.
+Users who can tolerate a 5 seconds delay from modifying a user's permissions to having them take effect through Admin Router are recommended to enable this feature.
+
```

Comment:
This section includes a lot of background info. on why we decided to make this change. Is it useful for customers to have all this info.? Some of it seems like implementation details, which might be more appropriate for internal discussion. It would be best to include only prescriptive info., i.e., what customers need to do in particular scenarios. I don't know enough about the issue, so perhaps all the info. is relevant. Please review and edit, if appropriate.